knative-operator/go.mod

```diff
@@ -0,0 +1,39 @@
+module github.com/openshift-knative/serverless-operator/knative-operator
+
+go 1.14
+
+require (
+	github.com/appscode/jsonpatch v0.0.0-20190108182946-7c0e3b262f30
+	github.com/coreos/prometheus-operator v0.29.0
+	github.com/go-logr/logr v0.1.0
+	github.com/gophercloud/gophercloud v0.6.0 // indirect
+	github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79 // indirect
+	github.com/manifestival/controller-runtime-client v0.2.0-0.1.12
+	github.com/manifestival/manifestival v0.6.1
+	github.com/openshift/api v0.0.0-20190927182313-d4a64ec2cbd8
+	github.com/openzipkin/zipkin-go v0.2.5 // indirect
+	github.com/operator-framework/operator-sdk v0.10.1
+	github.com/prometheus/client_golang v1.6.0
+	github.com/spf13/pflag v1.0.5
+	k8s.io/api v0.18.7-rc.0
+	k8s.io/apimachinery v0.19.0
+	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
+	knative.dev/operator v0.17.2
+	knative.dev/pkg v0.0.0-20200831162708-14fb2347fb77
+	knative.dev/serving v0.15.3
```

Comment:
0.17.3 is used, but we cannot update currently because we get clashes with client-go dependencies that we in turn cannot update because we have to use an old controller-runtime because of the way our webhooks are setup.

I plan to untangle all this (this PR is keeping all dependencies roughly the same as they used to be before) to be able to update everything to much more appropriate versions.

Reply:
Also, IMO, keeping the versions the same with `Gopkg.toml` is a better idea. ...unless we really really need to change them here because of some `go mod` problems.

Actually upgrading the dependency versions should be in a separate PR.

Reply:
Agree @aliok, that was the goal of this.